src/nav/browser.yml

```diff
@@ -124,6 +124,10 @@ pages:
         path: /docs/browser/new-relic-browser/browser-apis/log
       - title: noticeError
         path: /docs/browser/new-relic-browser/browser-apis/noticeerror
+      - title: pauseReplay
+        path: /docs/browser/new-relic-browser/browser-apis/pauseReplay/
```

Comment:
@ally-sassman I think the forward slash at the end of these two is preventing the navigation from keeping these in the left nav. When I click them in the left navigation pane, the correct file shows up, but the focus is lost in the left pane and it loads the generic list.

Reply:
@rhetoric101 I didn't think that the last `/` actually had an effect. It could be just a buggy build preview. But regardless, I'll add it anyway.